upi/openstack/04_control-plane.yaml

```diff
@@ -0,0 +1,46 @@
+# Required Python packages:
+#
+# ansible
+# openstacksdk
+# netaddr
+
+- hosts: all
+
+  vars:
+    # The provided filename will be concatenated with the Control Plane node
+    # name and its 1-indexed serial number.
```

Comment:
Please make it 0-indexed. That's what IPI as well as the UPI doc do currently.

Reply:
true, thanks. I was taking IPI as an example instead.